plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -619,8 +511,19 @@ public void testTimestamp(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_timestamp"))
                 .execute(getQueryRunner(), session, trinoCreateAsSelect(getSession(), "test_timestamp"))
                 .execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_timestamp"));
+    }
+
+    /**
+     * Read {@code TIMESTAMP}s inserted by MySql as Trino {@code TIMESTAMP}s
+     */
+    // TODO reuse these test cases in testTimestampFromTrino once we support timestamps with precisions other than 3 (https://github.com/trinodb/trino/issues/6910)
```

Comment:
"Reuse" is the wrong word here.

Since we separated these tests into their own method, I think this comment is probably unnecessary. We'll need to add more tests when we support more precisions, which is all that this comment is saying now.